src/tfvc/tfvc-extension.ts

```diff
@@ -44,6 +45,37 @@ export class TfvcExtension  {
         }
     }
 
+    /**
+     * This command runs the info command on the passed in itemPath and
+     * opens a web browser to the appropriate history page.
+     */
+    public async TfvcViewHistory(itemPath?: string): Promise<void> {
+        if (!this._manager.EnsureInitialized(RepositoryType.TFVC)) {
+            this._manager.DisplayErrorMessage();
+            return;
+        }
+
+        try {
+            if (!itemPath) {
+                this._manager.Telemetry.SendEvent(TfvcTelemetryEvents.OpenRepositoryHistory);
+                //Just display the history url of the entire repo
+                Utils.OpenUrl(this._manager.RepoContext.RemoteUrl + "_versionControl?_a=history");
```

Comment:
Does this assume that the RemoteUrl ends in "/"? Do we have a helper that join url paths like path.combine? That would be safer.